examples/async_summarize.py

```diff
@@ -0,0 +1,97 @@
+"""Copyright 2019 REV
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+import time
+from rev_ai import apiclient, JobStatus
+from rev_ai.models.asynchronous.summarization_job_status import SummarizationJobStatus
+from rev_ai.models.asynchronous.summarization_options import SummarizationOptions
+from rev_ai.models.nlp_model import NlpModel
+
+# String containing your access token
+access_token = "<your_access_token>"
+
+# Create your api client
+client = apiclient.RevAiAPIClient(access_token)
+
+# Submitting a job through a local file.
+#
+# job = client.submit_job_local_file("your_local_file_path",
+#                                    delete_after_seconds=2592000,
+#                                    language='en',
+#                                    summarization_config=SummarizationOptions(
+#                                        model=NlpModel.PREMIUM
+#                                    ))
+
+
+# Submitting a job with a link to the file you want transcribed
+# Change url to your url, custom_vocabularies is optional like above
+url = "https://www.rev.ai/FTC_Sample_1.mp3"
+job = client.submit_job_url(media_url=url,
+                            delete_after_seconds=2592000,
+                            language='en',
+                            summarization_config=SummarizationOptions(
+                                model=NlpModel.PREMIUM
+                            ))
```

Comment:
this is odd formatting style (the integration instead of 4 spaces is much larger) are you trying to follow the existing format?

Reply:
Most likely linter was complaining about the line length. I'll double check.

Reply:
In general - yes I was trying to follow existing formatting in other example files for consistency